CHANGELOG.md

```diff
@@ -197,6 +197,7 @@
 
 ### Fixes
 
+- `[expect]` Fix false positives when looking for undefined prop ([#8786](https://github.com/facebook/jest/issues/8786))
```

Comment:
this should be under master near the top of the file

Also, please mark it as breaking